README.md

```diff
@@ -1,36 +1,124 @@
+|EC2 instance type|m5.2xlarge|The EC2 instance type for Microsoft SQL server.|
+|Windows server AMI|/aws/service/ami-windows-latest/Windows_Server-2019-English-Full-SQL_2016_SP2_Standard|The latest Windows server 2019 with SQL 2016 Standard AMI.|
+|MSSQL Server version|13|MSSQL Server version. This is used to Change Auth mode from Windows only to SQL and Windows Auth For MSSQL server 2017 use number 14, for MSSQL server 2016 use number 13.|
+|RDS instance type|db.m5.large|Instance class of RDS instance.|
+|Database engine type|sqlserver-se|MS SQL engine type.|
+|Database engine version|13.00.5216.0.v1|MS SQL Engine version.|
+|Windows server and database username|dms_user|The database and instance admin account. Minimum 5 characters must begin with a letter and contain only alphanumeric or "_".|
+|Windows server and database password|Requires input|The password for instance user account. Minimum 8 characters, at least one of each of the following; uppercase, lowercase, number, and symbol character such as !@#$%^&*()<>[]{}|_+-=.|
+|Database name|dms_sample|Database name.|
 
-## Limitations
+   When completed, click *Next*
+1. [Configure stack options](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-console-add-tags.html) if desired, then click *Next*.
+1. On the review you screen, you must check the boxes for:
+   * "*I acknowledge that AWS CloudFormation might create IAM resources*" 
+   * "*I acknowledge that AWS CloudFormation might create IAM resources with custom names*"
+   * "*I acknowledge that AWS CloudFormation might require the following capability: CAPABILITY_AUTO_EXPAND*"
+
+   These are required to allow CloudFormation to create a Role to allow access to resources needed by the stack and name the resources in a dynamic way.
+1. Click *Create Stack* 
+1. Wait for the CloudFormation stack to launch. Completion is indicated when the "Stack status" is "*CREATE_COMPLETE*".
+   * You can monitor the stack creation progress in the "Events" tab.
+1. Note the *EC2SQLServerEip* and *RDSSQLEndpoint* displayed in the *Outputs* tab of the main stack. This can be used to access the EC2 host and RDS instance.
+
+### Populating database guide
+
+[Database guide](docs/database/README.md)
+
+### Limitations
 - MSSQL server 2017 doesnt support continues replication. The solution is using MSSQL server 2016 by default.
 
-## Cleanup
-All the resources are deployed using CLoudFormation nested stacks. 
-Deleting the main stack will delete all the resources created.
+### Remove the application
+
+To remove the application:
+
+1. Open the AWS CloudFormation Console
+1. Click the *aws-dms-sql-server* project, right-click and select "*Delete Stack*"
+1. Your stack will take some time to be deleted. You can track its progress in the "Events" tab.
+1. When it is done, the status will change from DELETE_IN_PROGRESS" to "DELETE_COMPLETE". It will then disappear from the list.
+
+### Making changes to the code and customization
```

Comment:
Do we need this? I think this message is specific to running front-end and I don't think we have any front-end here

Reply:
I have reword this to delete the stack, as these steps apply to clean up resources.